dashboard/utils.py

```diff
@@ -44,6 +44,15 @@ async def run(self, server):
         :param server: Asyncio GRPC server.
         """
 
+    @staticmethod
+    @abc.abstractclassmethod
+    def is_optional():
```

Comment:
should we name it `is_minimal` or something similar to be consistent with agent

Reply:
Is_minimal sgtm$

Reply:
Going with `is_minimal_module`